src/components/sbb-form-field/sbb-form-field.tsx

```diff
@@ -163,15 +163,17 @@ export class SbbFormField implements ComponentInterface {
     const value = this._errorElements.length
       ? this._errorElements.map((e) => e.id).join(',')
       : this._originalInputAriaDescribedby;
-    this._input?.setAttribute('aria-describedby', value);
+    value && this._input?.setAttribute('aria-describedby', value);
   }
 
   public render(): JSX.Element {
     return (
       <div class="sbb-form-field__space-wrapper">
         {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions */}
         <div onClick={(): void => this._input?.focus()} class="sbb-form-field__wrapper">
-          <slot name="prefix"></slot>
+          <div class="sbb-form-field__slot">
```

Comment:
The outer div (sbb-form-field__wrapper), has display:flex, and a gap, which was applied also to the slot's content (because by default the slot has display:contents) causing unwanted spacing between the arrow and the toggle. The slot doesn't allow classes unfortunately

Reply:
If you are sure that this does not have side effects, it's ok I think.

But I guess it's possible to apply classes, you just have to change display to flex of the <slot> element. not? But at the end it would be the same only with one element less in the DOM :)